src/radio/styled-components.js

```diff
@@ -137,7 +137,8 @@ export const Root = styled<StylePropsT>('label', props => {
     alignItems: 'center',
     cursor: $disabled ? 'not-allowed' : 'pointer',
     marginTop: sizing.scale200,
-    marginBottom: $hasDescription ? null : sizing.scale200,
+    marginLeft: ($align !== 'horizontal') ? sizing.scale200: null,
```

Comment:
Thanks for the contribution. When inspecting the screenshot changes [here](https://github.com/eekbatani/baseweb/pull/1/files?short_path=19699b9#diff-19699b9cc01a7b617c8f43c7dd260e60) There should not be margin-left applied when the alignment is vertical. Should the conditional on this line be inverted?

Reply:
Also, the first radio still shouldn't have left margin in horizontal alignment. Could you apply `marginRight` instead? Could also be `:first-child` rule but that would rely on no additional wrapper markup.

Reply:
You're right, I'll make the changes.